features/ignored_slots.feature

```diff
@@ -0,0 +1,54 @@
+Feature: ignored slots
+  Scenario: check ignored slots aren't removed on failover/switchover
+    Given I start postgres1
+    Then postgres1 is a leader after 10 seconds
+    And there is a non empty initialize key in DCS after 15 seconds
+    When I issue a PATCH request to http://127.0.0.1:8009/config with {"loop_wait": 2, "ignore_slots": [{"name": "unmanaged_slot_0", "database": "postgres", "plugin": "test_decoding", "type": "logical"}, {"name": "unmanaged_slot_1", "database": "postgres", "plugin": "test_decoding"}, {"name": "unmanaged_slot_2", "database": "postgres"}, {"name": "unmanaged_slot_3"}], "postgresql": {"parameters": {"wal_level": "logical"}}}
+    Then I receive a response code 200
+    # Make sure the wal_level has been changed.
+    When I shut down postgres1
+    And I start postgres1
+    Then postgres1 is a leader after 10 seconds
+    And "members/postgres1" key in DCS has role=master after 3 seconds
+    # Make sure Patroni has finished telling Postgres it should be accepting writes.
+    And postgres1 role is the primary after 20 seconds
+    And Response on GET http://127.0.0.1:8009/config contains ignore_slots after 10 seconds
```

Comment:
The `/config` endpoint is more or less proxy to the `/config` key in DCS. Since the [PATCH returned 200](https://github.com/zalando/patroni/pull/1742/files#diff-ff1c3e9e4d3e4c711f4d2c6f500b4ea23d93fe454b02ab5ebb86eeddc7771f26R6-R7), values were persisted in the DCS.

Reply:
We actually copied that assertion from the first test in `features/standby_cluster.feature`, but it does seem good to move it up to be part of the assertions that follow the PATCH, so we're changing that.